R/use-dittodb.R

```diff
@@ -31,7 +31,8 @@
 #' }
 use_dittodb <- function(path = ".") {
   if (!("DESCRIPTION" %in% dir(path))) {
-    stop(path, " is not an R package directory", call. = FALSE)
+    error_msg <- glue::glue("{path} is not an R package directory.")
+    rlang::abort(error_msg)
```

Comment:
```suggestion
    rlang::abort(glue("{path} is not an R package directory."))
```

Slightly simpler, what do you think?